.build/.build.csproj

```diff
@@ -14,7 +14,6 @@
         <PackageReference Include="JetBrains.ReSharper.CommandLineTools" ExcludeAssets="All" />
         <PackageReference Include="GitVersion.Tool" ExcludeAssets="All" />
         <PackageReference Include="ReportGenerator" ExcludeAssets="All" />
-        <PackageReference Include="JetBrains.ReSharper.CommandLineTools" ExcludeAssets="All" />
```

Comment:
I removed this because it was a duplicate line and the build tools complained about it. (See line 14.)